blocks/extensions/monitoring.py

```diff
@@ -139,3 +139,23 @@ def do(self, callback_name, *args):
             self.add_records(self.main_loop.log,
                              self._buffer.get_aggregated_values().items())
             self._buffer.initialize_aggregators()
+
+
+class MonitoredQuantity(object):
+    def __init__(self):
+       pass
+
+
+class CrossEntropy(MonitoredQuantity):
+    def __init__(self, **kwargs):
+        super(CrossEntropy, self).__init__(**kwargs)
+        self.total_CrossEntropy, self.examples_seen = 0, 0
```

Comment:
I would suggest reading [this](https://www.python.org/dev/peps/pep-0008/#naming-conventions) to learn about python naming conventions. 

In brief, it should be `total_cross_entropy`.